src/test/java/Pages/MainPage.java

```diff
@@ -0,0 +1,33 @@
+package Pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class MainPage {
+
+    WebDriver driver;
+
+    public MainPage(WebDriver driver){
+        this.driver=driver;
+    }
+
+    By addUserLink = By.xpath("//a[@class='btn btn-success']");
+    By viewDetailsLink = By.xpath("//*[@id=\"users-table-body\"]/tr[1]/td[4]/button[1]");
```

Comment:
I suggest not using xpath by array indexes. You can try to find button with text.